ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/TLSOnePointThreeIT.scala

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.suites
+
+import com.daml.ledger.api.testtool.infrastructure.Allocation.{NoParties, allocate}
+import com.daml.ledger.api.testtool.infrastructure.participant.ParticipantTestContext
+import com.daml.ledger.api.testtool.infrastructure.{Endpoint, LedgerTestSuite}
+import com.daml.ledger.api.tls.TlsVersion
+import com.daml.ledger.api.tls.TlsVersion.TlsVersion
+import com.daml.ledger.api.v1.ledger_identity_service.LedgerIdentityServiceGrpc.LedgerIdentityServiceBlockingStub
+import com.daml.ledger.api.v1.ledger_identity_service.{
+  GetLedgerIdentityRequest,
+  LedgerIdentityServiceGrpc,
+}
+import com.daml.ledger.resources.{ResourceContext, ResourceOwner}
+import io.grpc.StatusRuntimeException
+import io.grpc.netty.NettyChannelBuilder
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.util.{Failure, Success, Try}
+
+/** Verifies that the given participant server correctly handles TLSv1.3 only mode.
+  *
+  * It works by creating and exercising a series of client service stubs, each over different TLS version.
+  * Only TLSv1.3 connection is expected to succeed.
+  * Connections over lower TLS versions are expected to fail.
+  */
+final class TLSOnePointThreeIT extends LedgerTestSuite {
```

Comment:
@mziolekda @pbatko-da : what do you think about also adding a test that ensures that by TLS 1.0 and TLS 1.1 connections are never allowed, which is the current recommendation. See e.g., https://docs.microsoft.com/en-us/lifecycle/announcements/transport-layer-security-1x-disablement

Reply:
This might be already covered by this test depending what exactly you have in mind.

To recap, this test checks that:
1) clients on TLSv1.3 succeed,
2) clients on TLS 1.2, 1.1 and 1.0 fail. 
(It follows that for this test to pass the server must be started in TLSv1.3 only mode.)

Do you possibly mean to add a test case that only ensures that clients on TLS 1.0 and 1.1 always fail? This would partially duplicate this test case but having just these two cases separate would mean it could be a default test case of Ledger AP Test Tool (as opposed to being optional that needs to be explicitly `--include`d or `--additional`ed) .

Reply:
Simon, we have decided to create a test that verifies minimum client version of 1.3. We felt that it is the only one that makes sense. If one is serious about security, should set the server to a minimum of 1.3 and that's what this test asserts.

Reply:
@mziolekda : TLS 1.3 was defined in August 2018, so is very recent in enterprise terms. TLS 1.2 has not been deprecated. I would thus not be surprised that there are users of our software that want to use TLS 1.2 instead of TLS 1.3. For these, it is very important that the actually deprecated TLS 1.0 and 1.1 versions are not accepted. See https://en.wikipedia.org/wiki/Transport_Layer_Security#TLS_1.2 for background.

So, I believe we either decide to not support TLS 1.2 at all, or we do test that if TLS 1.2 is used, the 1.0 and 1.1 versions are not supported.

@mziolekda : how would you like to proceed?